walkthroughs/week-1-web-development/examples/stanley/pom.xml

```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```

Comment:
Looks like your Jetty updates made it in this pull request as well - no worries for this request, but try to keep them separate

Reply:
I think that they said the updates should be made to all of the pom.xml walkthrough files but I could be wrong.

Reply:
Ah, okay, I didn't realize these were different than in the previous pull request, sounds good! Just wanted to make sure this wasn't a branching issue :)